### ISSUE TITLE
Сделать новую, более верную и доступную HTML-разметку

### ISSUE THREAD

**Issue by pepelsbey** [web-standards-ru/weblind.ru#40]
Это будет первым шагом в перевёрстке проекта, завёл для этого отдельную ветку `markup` и там уже есть [первые три HTML файла](https://github.com/web-standards-ru/weblind.ru/tree/markup/src) с черновой разметкой. Пока специально без классов, основной контент пока пунктирно, но идея вырисовывается.

**Comment by arslan-akh** [comment]
А есть какие-либо предпочтения по именованию классов CSS и технологиям верстки?

**Comment by discoder-git** [comment]
Приветствую, коллеги. Подскажите пожалуйста, знающие люди, а главная навигация проекта (18 строка) под блоком с основным контентом (13 строка) - это так по стандартам нужно? Дело в том, что я его всё в хедер помещаю (8 строка). Вот хотел прояснить этот вопрос - куда (с точки зрения построения современной разметки) его помещать? (если без привязки к дизайну)

**Comment by pepelsbey** [comment]
@arslan-akh сейчас специально не думаю про вёрстку и классы, только про разметку.

**Comment by pepelsbey** [comment]
@discoder-git заходите лучше в [Слак Веб-стандартов](http://slack.web-standards.ru) с этим вопросом, там есть канал #html. Хочется здесь ровно по делу.

**Comment by arslan-akh** [comment]
Какой план действий? Вот сейчас есть ветка markup, что требуется для этой ветки на данном этапе? Заполнить контентом без стилей?

**Comment by pepelsbey** [comment]
Да, сейчас хочется заполнить всё существующим содержимым без стилей и классов. Чисто и хорошо, потестировать и перейти ко второму шагу — вёрстке.

**Comment by nikolai-shabalin** [comment]
Привет. Я в #54 сомневаюсь в теге `abbr` для W3C, HTML, CSS текстов. Подскажите правильно ли я всё сделал.

Также я клавишные сочетания обернул в `<kbd>`, судя по [htmlbook](http://htmlbook.ru/html/kbd) я всё верно сделал, но что-то мучают сомнения.

Ещё я тексты переношу на длине в 120 символов, так как длинный текст гораздо сложнее править. Если 120 символов годно, то везде поправлю и дополню свой PR. Можно и не 120 символов, но какое-то ограничение точно нужно.

**Comment by pepelsbey** [comment]
@nikolai-shabalin `<abbr>` и `<kbd>` вполне уместные теги для упомянутых случаев. Переносить тексты лучше доверить настройкам редактора, а не хардкодить это. Просто иногда удобно увидеть код построчно, а иногда в узкой колонке с переносами. Хардкод намертво прибивает один вариант. Написал ревью для #54.
